src/components/calendar/Calendar.js

```diff
@@ -2592,9 +2592,9 @@ export class Calendar extends Component {
     }
 
     renderDayNames(weekDays) {
-        const dayNames = weekDays.map(weekDay =>
+        const dayNames = weekDays.map( (weekDay, index) =>
         (
-            <th key={weekDay} scope="col">
+            <th key={index} scope="col">
```

Comment:
Using index for Keys is an Antipattern as it can lead to errors when weekdays change, as the indexes won't change. https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

```suggestion
            <th key={`${weekDay}-${index}`} scope="col">
```